staging/src/k8s.io/cloud-provider/volume/helpers/zones.go

```diff
@@ -118,9 +118,12 @@ func SelectZonesForVolume(zoneParameterPresent, zonesParameterPresent bool, zone
 
 		// pick node's zone for one of the replicas
 		var ok bool
-		zoneFromNode, ok = node.ObjectMeta.Labels[v1.LabelFailureDomainBetaZone]
+		zoneFromNode, ok = node.ObjectMeta.Labels[v1.LabelTopologyZone]
```

Comment:
@msau42 I have made change to the cloud-provider so that this can handle both GA version and beta version label in the node. This should be safer now.